src/modules/Settings/Organization/OrganizationImport.ts

```diff
@@ -0,0 +1,76 @@
+import { IInputProps } from "../../../shared/Forms/Form";
+import { ColumnDef } from "@tanstack/react-table";
+import fileUploads from "../../../services/fileuploads.json";
+export type Person = {
+  name: string;
+  age: number;
+  country: string;
+};
+
+export const organizationForm: IInputProps[] = [
+	{
+	  label: "Full Name",
+	  placeholder: "Your Full Name",
+	  type: "input",
+	  value: [],
+	  isRequired: true,
+	},
+	
+	{
+	  label: "Phone Number",
+	  placeholder: "09012345678",
+	  type: "input",
```

Comment:
the placeholder should be 'your phone number' or some words, remove the hardcoded value

Reply:
Done